<launch>
   <node name = "path_follower" type="path_follower.py" pkg="lab3" output="screen">
      <param name="pose_topic" type="string" value="/sim_car_pose/pose"/>
      <param name="plan_lookahead" type="double" value="1"/>
      <param name="translation_weight" type="double" value="1.0"/>
      <param name="rotation_weight" type="double" value=".5"/>
      <param name="kp" type="double" value="0.7"/>
      <param name="kd" type="double" value="0.3"/>
      <param name="ki" type="double" value="0.0"/>
      <param name="error_buff_length" type="double" value="10"/>
      <param name="speed" type="double" value="2"/>   
      <param name="handoff_thresh" type="double" value="10.0"/>
      <param name='bag_path' type = 'string' value='/home/nvidia/catkin_ws/src/lab3/bags/full_car_plan.bag'/>

   </node>
</launch>
